addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.xml

```diff
@@ -14,7 +14,7 @@
                         </t>
                     </label>
                 </div>
-                <t t-if="!suggestedRecipientInfo.partner">
+                <t t-if="!suggestedRecipientInfo.partner and suggestedRecipientInfo.thread">
```

Comment:
thread is not supposed to become empty, how comes that happen?

Reply:
After investigation: this was indeed not the correct fix.
It was just a consequence of obsolete suggestedRecipientInfo being unlinked from its related thread but not deleted.